app/config/variables.php

```diff
@@ -144,9 +144,18 @@
             ],
             [
                 'name' => '_APP_SYSTEM_EMAIL_ADDRESS',
-                'description' => 'This is the sender email address that will appear on email messages sent to developers from the Appwrite console. The default value is \'team@appwrite.io\'. You should choose an email address that is allowed to be used from your SMTP server to avoid the server email ending in the users\' SPAM folders.',
+                'description' => 'This is the sender email address that will appear on email messages sent to developers from the Appwrite console. The default value is \'noreplay@appwrite.io\'. You should choose an email address that is allowed to be used from your SMTP server to avoid the server email ending in the users\' SPAM folders.',
```

Comment:
```suggestion
                'description' => 'This is the sender email address that will appear on email messages sent to developers from the Appwrite console. The default value is \'noreply@appwrite.io\'. You should choose an email address that is allowed to be used from your SMTP server to avoid the server email ending in the users\' SPAM folders.',
```